core/rawdb/accessors_chain.go

```diff
@@ -1597,6 +1597,77 @@ func DeleteCreatedUTXOKeys(db ethdb.KeyValueWriter, blockHash common.Hash) {
 	}
 }
 
+func WriteCreatedCoinbaseLockupKeys(db ethdb.KeyValueWriter, blockHash common.Hash, keys [][]byte) error {
+	protoKeys := &types.ProtoKeys{Keys: make([][]byte, 0, len(keys))}
+	protoKeys.Keys = append(protoKeys.Keys, keys...)
+
+	data, err := proto.Marshal(protoKeys)
+	if err != nil {
+		db.Logger().WithField("err", err).Fatal("Failed to rlp encode utxo")
+	}
+	return db.Put(createdCoinbaseLockupsKey(blockHash), data)
+}
+
+func ReadCreatedCoinbaseLockupKeys(db ethdb.Reader, blockHash common.Hash) ([][]byte, error) {
+	// Try to look up the data in leveldb.
+	data, _ := db.Get(createdCoinbaseLockupsKey(blockHash))
+	if len(data) == 0 {
+		return nil, nil
+	}
+	protoKeys := new(types.ProtoKeys)
+	if err := proto.Unmarshal(data, protoKeys); err != nil {
+		return nil, err
+	}
+	return protoKeys.Keys, nil
+}
+
+func DeleteCreatedCoinbaseLockupKeys(db ethdb.KeyValueWriter, blockHash common.Hash) {
+	if err := db.Delete(createdCoinbaseLockupsKey(blockHash)); err != nil {
+		db.Logger().WithField("err", err).Fatal("Failed to delete created coinbase lockup keys")
+	}
+}
+
+func WriteDeletedCoinbaseLockups(db ethdb.KeyValueWriter, blockHash common.Hash, deletedLockups map[[47]byte][]byte) error {
```

Comment:
why 47? also can you make this a parameter, so its easier to know what this is

Reply:
You actually can't make a map of []]byte with a parameter as the size, I've tried. But it is the size of the keys

Reply:
you can have a const for sure, we do that in the codebase, i understand if we cannot do a var

Reply:
you can have a const for sure, we do that in the codebase, i understand if we cannot do a var

Reply:
Fixed, it's const now